dev-lang/gnucobol/gnucobol-2.2.recipe

```diff
@@ -0,0 +1,123 @@
+SUMMARY="A free COBOL compiler"
+DESCRIPTION="GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler."
+HOMEPAGE="https://www.gnu.org/software/gnucobol/"
+COPYRIGHT="2001-2017 Free Software Foundation, Inc
+	Keisuke Nishida
+	Roger While
+	Ron Norman
+	Simon Sobisch
+	Edward Hart
+	Brian Tiffin
+	Sergey Kashyrin
+	Dave Pitts"
+LICENSE="GNU GPL v3
+	GNU LGPL v3
+	GNU FDL v1.3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/open-cobol/files/gnu-cobol/$portVersion/gnucobol-$portVersion.tar.gz"
+CHECKSUM_SHA256="925838decd65864b2aa3a4bf1385ce4bc708b942e05e8406945a730d7aab32cb"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gnucobol$secondaryArchSuffix = $portVersion
+	cmd:cobc$commandSuffix = $portVersion
+	cmd:cobcrun$commandSuffix = $portVersion
+	lib:libcob$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libdb$secondaryArchSuffix
+	lib:libgmp$secondaryArchSuffix
```

Comment:
Is there an optional / variant definition for haikoports? If yes then it would be reasonable to have the libintl, curses and bdb parts marked as those).
Note: for 3.1 you'll likely add cjson (or put it into "libcob" to get it included in libcob) and libxml2, too.

Reply:
I don't see such requirements for ArchLinux: https://www.archlinux.org/packages/community/x86_64/gnucobol/
so we'd rather keep things simple.

Reply:
I'd say that the archlinux package would be ideally also use the [`optdepends` array][https://wiki.archlinux.org/index.php/PKGBUILD#optdepends] ;-)

Reply:
Would need to re-check what configure is looking for, if libintl, ncurses are being looked for we could work on this, but in this version is didn't seem to be a requirement (eg can build without it)

Reply:
ncursesw/ncurses + libintl are looked for, if not available they are not included; bdb is always looked for and if not available configure stops. All of those can be forced to be enabled/disabled with --enable-stuff/--disable-stuff (see `configure --help`).

Reply:
Aside from cjson all things should be ok to build this ...

Reply:
Builds OK, checking the tests atm and will send you the results over email @GitMensch